osu.Game.Rulesets.Diva/Difficulty/DivaDifficultyCalculator.cs

```diff
@@ -18,9 +18,23 @@ public DivaDifficultyCalculator(Ruleset ruleset, WorkingBeatmap beatmap)
         {
         }
 
-        protected override DifficultyAttributes CreateDifficultyAttributes(IBeatmap beatmap, Mod[] mods, Skill[] skills, double clockRate)
-        {
-            return new DifficultyAttributes(mods, skills, 0);
+        protected override DifficultyAttributes CreateDifficultyAttributes(IBeatmap beatmap, Mod[] mods, Skill[] skills, double clockRate) {
+            double od = beatmap.BeatmapInfo.BaseDifficulty.OverallDifficulty;
+
+            double difficulty = 1;
+            
```

Comment:
```suggestion
            //TODO: This will need to be rewritten once we start work on #9
```

Maybe add a warning on this code